### DISCUSSION TITLE
Interactive Pivot Table in Report

### DISCUSSION THREAD

**Discussion by mfein2** [datapane/datapane#136] [Help]
Does anyone know of a way to insert an interactive pivot table within a datapane report?  I currently use pivottable.js to make these and being able to embed one would be great.

**Comment by mands** [comment]
Hi @mfein2 - currently there isn't an easy way to work with pivot tables - you could look at using `dp.HTML` to manually embed a `pivottable.js` table for now.

We've looked into using `pivotable.js` before but found it didn't integrate well with our existing `dp.DataTable` component, but are looking at other ways we can add pivot functionality to `dp.DataTable` to address this.

HTH!
